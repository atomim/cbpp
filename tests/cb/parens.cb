#define root(a) sqrt(a)

addtext "" + root(getKey())
addtext "" + root(asc(root(64)))
