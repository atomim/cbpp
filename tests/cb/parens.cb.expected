// #define root(a) sqrt(a)

addtext "" + sqrt(getKey())
addtext "" + sqrt(asc(sqrt(64)))
